pkg/services/ngalert/api/tooling/definitions/alertmanager_validation.go

```diff
@@ -65,3 +100,14 @@ func (r *Route) Validate() error {
 	}
 	return r.validateChild()
 }
+
+func (mt *MuteTimeInterval) Validate() error {
+	s, err := yaml.Marshal(mt.MuteTimeInterval)
```

Comment:
For mute timings we use the raw prometheus type, which has its own validation code in the serializer, living in the prometheus project.

There is a lot of validation code - a few hundred lines. Just run the serializer, don't try to duplicate all of that.